_es/tour/tuples.md

```diff
@@ -10,5 +10,82 @@ previous-page: inner-classes
 
 ---
 
-(this section of the tour has not been translated yet. pull request
-with translation welcome!)
\ No newline at end of file
+En Scala, una tupla es un valor que contiene un número fijo de elementos,
+cada uno de ellos puede ser de distinto tipo. Las tuplas son inmutables.
+
+Las tuplas son especialmente útiles para retornar múltiples valores desde 
+un método.
+
+Una tupla con dos elementos puede ser creada del siguiente modo:
+
+```tut
+val ingredient = ("Sugar", 25)
+```
+
+Esta instrucción crea una tupla que contiene un elemento de tipo `String` 
+y un elemento de tipo `Int`.
+
+El tipo de la tupla `ingredient` se infiere que es`(String, Int)`, lo cual es
+una abreviatura de `Tuple2[String, Int]`.
+
+Para representar tuplas, Scala utiliza una serie de clases: `Tuple2`, `Tuple3`, 
+etc., hasta `Tuple22`.
+Cada clase tiene tantos parámetros como número de elementos.
+
+## Accediendo a los elementos
+
+Una forma de acceder a los elementos de una tupla es por posición.
+Los elementos concretos se llaman `_1`, `_2`, y así sucesivamente.
+
+```tut
+println(ingredient._1) // Sugar
+println(ingredient._2) // 25
+```
+
+## Pattern matching en tuplas
```

Comment:
Choose between using reconocimiento de patrones and pattern matching and be consistent, I guess

Reply:
I prefer to keep it in English, but there are some previous translations with this term translated.